tests/TestRunner/app/ObjCConstructors.js

```diff
@@ -79,46 +79,52 @@ describe("Constructing Objective-C classes with new operator", function () {
     });
 
     describe("Swift-style initializers", () => {
-      it("should work", () => {
-        let obj = new NSObject();
-        expect(obj).toEqual(jasmine.any(NSObject));
-        expect(obj.retainCount()).toBe(1);
-      });
-
-      it("should support parameters", () => {
-        let arr = new NSArray({array : [ 1, 2, 3 ]});
-        expect(arr).toEqual(jasmine.any(NSArray));
-        expect(arr.count).toEqual(3);
-      });
-
-      it("should support even more complex parameters", () => {
-        let alertView = new UIAlertView({
-          title : "About",
-          message : "NativeScript Team",
-          delegate : null,
-          cancelButtonTitle : "OK",
-          otherButtonTitles : null
+        it("should work", () => {
+            let obj = new NSObject();
+            expect(obj).toEqual(jasmine.any(NSObject));
+            expect(obj.retainCount()).toBe(1);
+        });
+
+        it("should support parameters", () => {
+            let arr = new NSArray({
+                array: [1, 2, 3]
+            });
+            expect(arr).toEqual(jasmine.any(NSArray));
+            expect(arr.count).toEqual(3);
+        });
+
+        it("should support even more complex parameters", () => {
+            let alertView = new UIAlertView({
+                title: "About",
+                message: "NativeScript Team",
+                delegate: null,
+                cancelButtonTitle: "OK",
+                otherButtonTitles: null
+            });
+            expect(alertView.title).toEqual("About");
+            expect(alertView.message).toEqual("NativeScript Team");
+            expect(alertView.buttonTitleAtIndex(0)).toEqual("OK");
+        });
+
+        it("should support void initializers", () => {
+            let transform = new TNSCInterface({
+                identity: void 0
```

Comment:
Shouldn't this be `empty: void 0`, seeing as how the initializer is named `initWithEmpty`?